### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Smartiut**
man readmd ozgartrb yaxwi iw qilmadm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#Olleh Dlrow
```
Comment (Nuruddinjr) [inline README.md:1]: barbr creativemas 

Ahror yaxshi qiganakan

**Comment by Nuruddinjr** [comment]
uzr Nodir

**Comment by Smartiut** [comment]
hcqsi yo, kettumi?

**Comment by Nuruddinjr** [comment]
kettu kettu

**Comment by Smartiut** [comment]
fotiha berin  endiii hoji okaaa

**Comment by Nuruddinjr** [comment]
ie nmaga manda endi comment keldi?

**Pull request closed without merging**